client/clothing.lua

```diff
@@ -846,8 +846,8 @@ function ToggleClothing(whic, extra)
 					LastEquipped[which] = Cur
 					SetPedComponentVariation(Ped, Toggle.Drawable, Table, 0, 0)
 					if Toggle.Table.Extra then
-						Extras = Toggle.Table.Extra
-						for _, v in pairs(Extras) do
+						local Extras = Toggle.Table.Extra
+						for k, v in pairs(Extras) do

```

Comment:
Please change the k in both instances back to a _ other than that should be good